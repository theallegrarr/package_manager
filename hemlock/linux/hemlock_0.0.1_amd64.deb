version https://git-lfs.github.com/spec/v1
oid sha256:07a0871f45f58e59ad0124b638b60764689211caa989b955bc0d47d6257ee54a
size 81449252
