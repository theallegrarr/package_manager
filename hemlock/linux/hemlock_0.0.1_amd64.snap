version https://git-lfs.github.com/spec/v1
oid sha256:cfd34c2c1a4202afe7bb615806b920dc4d1fedafabaea4d04e50020ebfebabb6
size 103276544
